res/openvpn-gui-res-it.rc

```diff
@@ -104,6 +105,20 @@ BEGIN
     PUSHBUTTON "&Nascondi", ID_HIDE, 100, 190, 50, 14
 END
 
+/* Message Window.
+ * This dialog needs no translation unless a different font instead
+ * of the specified Segoe UI (or its equivalent) is preferred.
+ */
```

Comment:
As the comment says this whole dialog doesn't need translation nor copying to this file, unless you want a different font. That said, I can remove it. No point in translating anything here as all text including the header are replaced at run time by what is received from the server.

Reply:
Fixed